corehq/sql_db/connections.py

```diff
@@ -117,20 +122,19 @@ def get_engine(self, engine_id=DEFAULT_ENGINE_ID):
         Get an engine by ID. This should be a unique field identifying the connection,
         e.g. "report-db-1"
         """
-        return self._get_or_create_helper(engine_id).engine
+        return self.get_session_helper(engine_id).engine
 
-    def get_load_balanced_read_db_alias(self, engine_id, default=None):
+    def get_load_balanced_read_db_alias(self, engine_id):
```

Comment:
previously this would return either and `engine_id` or a DB alias which would then later be used to get a session helper. It now always returns a DB alias.